src/core/Config.h

```diff
@@ -90,17 +90,17 @@ enum UIScaleFilter {
 namespace {
```

Comment:
No anonymous namespace in headers! (Also they might not do what you expect in source files due to unity builds.)